src/Progress/CircularProgress.js

```diff
@@ -53,18 +67,37 @@ export const styleSheet = createStyleSheet('MuiCircularProgress', theme => ({
 }));
 
 function CircularProgress(props) {
-  const { classes, className, size, ...other } = props;
+  const { classes, className, size, mode, value, min, max, ...other } = props;
   const radius = size / 2;
+  const rootProps = {};
+  const svgClasses = classNames(classes.svg, {
+    [classes.indeterminateSvg]: mode === 'indeterminate',
+  });
+
+  const circleClasses = classNames(classes.circle, {
+    [classes.indeterminateCircle]: mode === 'indeterminate',
+    [classes.determinateCircle]: mode === 'determinate',
+  });
+
+  const styles = { circle: {} };
+  if (mode === 'determinate') {
+    const relVal = getRelativeValue(Math.round(value), min, max);
```

Comment:
`relValue` should always be between 0-1. I'll also change the calculation in `getRelativeValue` to reflect that correctly (ex. doesnt work for min 4, max 10, value 8).

Actually i think that maybe there is no need for the round at all?
If we want to support only round numbers then round should be done before calling `getRelativeValue`. Same goes for min/max.
But technically, it will work with positive fraction numbers as well.